OAuth/ResourceOwner/FiwareResourceOwner.php

```diff
@@ -86,7 +85,7 @@ public function getUserInformation(array $accessToken, array $extraParameters =
         }
 
         $response = $this->getUserResponse();
-        $response->setData($content->getContent());
+        $response->setData((string) $content->getBody());
```

Comment:
This `->getBody(true)` is not existing in fact in the code: https://github.com/guzzle/psr7/blob/master/src/MessageTrait.php#L123